packages/features/landing_page_screen/lib/src/landing_page.dart

```diff
@@ -80,7 +85,15 @@ class _LandingPageState extends ConsumerState<LandingPage> {
       backgroundColor: lightColorScheme.primary,
       appBar: AppBar(
         title: Text(widget.title),
-        actions: const [MainOverflowMenu()],
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.search_rounded, color: Colors.white), 
```

Comment:
please use custom colors we have in light theme. i think: lightColorTheme.onSurface is white color